test/test_infra.py

```diff
@@ -73,6 +77,23 @@ def test_pep8(self):
         report += ["\n-- Original flake8 reports:"] + flake8_issues
         self.fail("\n".join(report))
 
+    def test_pep257(self):
+        python_filepaths = get_python_filepaths(include_tests=False)
```

Comment:
Why is `test` directory not included in this test? Just curious for my own understanding.

Reply:
Because by convention is ok in this project to not have dosctrings on every test method.